nemo_skills/inference/prompt/openai/chat.yaml

```diff
@@ -0,0 +1,24 @@
+few_shot_examples:
+  template: "Example question:\n{question}\n\n{context}Example solution:\n{generation}\n\n\n\n\n\n"
+
+system: |-
+  You're an expert mathematician. Your goal is to solve the math problem below.
+  To achieve this you always follow these steps:
+  1. Start by carefully analyzing the given problem.
+  2. Write DETAILED step-by-step solution.
+  3. Put the final answer inside \boxed{{}}.
+
+user: |-
+    Here are some examples of questions and solutions followed by a new question that you need to solve.
+
+    {examples}Question:
+    {question}
+
+    {context}
+
+prompt_template: |-
+  <system_start>{system}<system_end>
```

Comment:
Can you please add a comment here for clarity that these are not used directly but will be parsed out by our api?